app/src/main/java/com/github/swent/echo/compose/association/AssociationListScreen.kt

```diff
@@ -43,39 +34,23 @@ fun AssociationListScreen(
 fun AssociationListElement(
     association: Association,
     onRowClicked: (Association) -> Unit,
-    onAssociationClicked: (Association) -> Unit,
-    eventsFilter: List<Association>
 ) {
     val boxInsidePadding = 5.dp
     val tonalElevation = 5.dp
-    val paddingSelectedIcon = 5.dp
     ListItem(
         modifier =
             Modifier.clickable { onRowClicked(association) }
                 .testTag("association_list_${association.name}"),
         overlineContent = { Spacer(modifier = Modifier) }, // used to align the icon and the button
         headlineContent = {},
         leadingContent = {
-            TextButton(
-                onClick = { onAssociationClicked(association) },
-                modifier = Modifier.testTag("association_name_button_${association.name}")
-            ) {
-                Text(
-                    text = association.name,
-                    textAlign = TextAlign.Center,
-                    modifier = Modifier.align(Alignment.CenterVertically).padding(boxInsidePadding)
-                )
-            }
-        },
-        trailingContent = {
-            if (eventsFilter.contains(association)) {
-                Icon(
-                    Icons.Filled.Done,
-                    "Selected",
-                    Modifier.padding(paddingSelectedIcon)
-                        .testTag("selected_icon_${association.name}")
-                )
-            }
+            Text(
+                text = association.name,
+                textAlign = TextAlign.Center,
+                modifier =
+                    Modifier.padding(boxInsidePadding)
+                        .testTag("association_name_button_${association.name}")
+            )
```

Comment:
You could move this inside `headlineContent` avoids having `headlineContent = {}`...

Reply:
Right ! it's more logical